lib/restrictions.js

```diff
@@ -53,10 +59,30 @@ export async function insert({restriction} = {}) {
   const collection = database.collections['resource-restriction-restriction'];
   const now = Date.now();
   const meta = {created: now, updated: now};
+  if(!restriction.id) {
+    throw new BedrockError(
+      `Id is required.`,
+      'ValidationError', {
+        public: true,
+        httpStatusCode: 400
+      });
+  }
   let record = {
     meta,
     restriction
   };
+  if(restriction.method === 'limitOverDuration') {
```

Comment:
So this check shouldn't happen here -- it's a layer violation. The `limitOverDuration` is an extension -- one that happens to come with the core module, but an extension nonetheless.

Reply:
Removed.